spec/visual/samples/dafny

```diff
@@ -87,36 +87,36 @@ set_ if_ as_ is_ int_ true_
 set' if' as' is' int' true'
 
 // Types
-        bool char int real string object object?
-        seq set map iset imap multiset
-        array array2 array10 array?
-        bv0 bv1 bv10 bv9
+bool char int real string object object?
+seq set map iset imap multiset
+array array2 array10 array?
```

Comment:
Other good examples to include here are the keywords `array2?` and `array10?`.

Reply:
added